src/main/java/org/folio/rest/impl/HoldingsStorageAPI.java

```diff
@@ -90,71 +90,19 @@ public void deleteHoldingsStorageHoldings(String lang,
   @Override
   public void getHoldingsStorageHoldings(
     int offset, int limit, String query, String lang,
-    Map<String, String> okapiHeaders,
+    RoutingContext routingContext, Map<String, String> okapiHeaders,
```

Comment:
Is this a change in the code generation that RAML Module Builder uses or is `routingContext` an optional parameter?

Reply:
The routingContext is parameter for paths in property generate_routing_context. This facility has been around for quite some time. See RMB-570

Reply:
Thanks, I wondered if there was a standard way of configuring this.